wasm-rpc-stubgen/src/rust.rs

```diff
@@ -476,10 +477,10 @@ fn generate_function_stub_source(
                 Span::call_site(),
             );
             let root_name = Ident::new(
-                &format!("{}_stub", def.source_package_name.name.to_snake_case()),
+                &format!("{}_client", def.source_package_name.name.to_snake_case()),
```

Comment:
Should not all these format!-d names coming from the `names` module instead?

Reply:
they should

Reply:
i will do another round on searching for them

Reply:
updated the PR with more naming things